frontend/src/components/elements/Metric/styled-components.ts

```diff
@@ -31,6 +31,19 @@ export const StyledTruncateText = styled.div(({ theme }) => ({
   fontFamily: theme.genericFonts.bodyFont,
   lineHeight: theme.lineHeights.normal,
   verticalAlign: "middle",
+  display: "flex",
```

Comment:
From the testing that I did yesterday / later today too, you need ellipses to appear by setting display to "inline-block" or "block". However, whenever I do that, it always makes it so the help icon doesn't fit.

Reply:
Just sent the fix for that [here](https://github.com/streamlit/streamlit/pull/6129/commits/f14bf7af89c02a467d7eef3009e184e7312ce5c3)!